Settings.py

```diff
@@ -1,18 +1,7 @@
-path_to_adBlock = r"your\path\here\Resources\adblocker\3.9_0"
-path_to_webdriver = r"your\path\here\Resources\windows chromedriver\chromedriver.exe"
+import Secure
 
-<<<<<<< HEAD
-yahoo_username = "anthonygrant414@yahoo.com"
-yahoo_password = "yellow613@G!24"
-=======
-yahoo_username = ""
-yahoo_password = ""
->>>>>>> 808fc79399c662936a4ddd8f81222d64fd535e17
+path_to_adBlock = Secure.path_to_adBlock + r"\Resources\adblocker\3.9_0"
+path_to_webdriver = Secure.path_to_webDriver + r"\Resources\windows chromedriver\chromedriver.exe"
```

Comment:
This is the line I couldn't pep8 format on my own. I can use black to see what it tells me.